carbonmark/components/pages/Project/Retire/Pool/RetireInputs.tsx

```diff
@@ -332,7 +328,11 @@ export const RetireInputs: FC<Props> = (props) => {
               message: t`Could not calculate Total Cost`,
             },
             validate: {
```

Comment:
ok this is wild, it's an invisible InputField used to add extra validations?

<img width="248" alt="image" src="https://github.com/KlimaDAO/klimadao/assets/88635679/168072dc-15c9-4987-811f-bbca0bb22c14">

If it's not too much trouble I would tear this out, and move the validations to the quantity input (`validate` object or callback) https://www.react-hook-form.com/api/useform/register/
If that's not feasible then we could use a useEffect which `watch`es the formstate and invokes `setError`